vms/platformvm/state/state_test.go

```diff
@@ -838,3 +839,108 @@ func TestStateAddRemoveValidator(t *testing.T) {
 		require.Equal(diff.expectedPublicKeyDiff, gotPublicKeyDiffs)
 	}
 }
+
+func TestParsedStateBlock(t *testing.T) {
+	require := require.New(t)
+
+	var blks []blocks.Block
+
+	{
```

Comment:
these scopes are to avoid overwriting previous entries right?

Reply:
Yeah, also to make it easier to delineate different types of blocks. We could definitely write this function without all these scopes. Since we'll be deleting it in v1.11.x, I'm inclined to keep it as-is